enacts/flex_fcst/cpt.py

```diff
@@ -102,3 +104,38 @@ def starts_list(
     start_dates = sorted(set(start_dates)) #finds unique dates in the case there are files with the same date due to multiple lead times
     start_dates = [i.strftime(format_out) for i in start_dates]
     return start_dates
+
+
+def read_pycptv2dataset(data_path):
+    for mm in (np.arange(11) + 1) :
```

Comment:
FYI this ends at November.

Don't bother fixing, because I'm rewriting this whole function. I continue to get hangs that appear to be caused by multiple callbacks calling `open_mfdataset` concurrently, so I'm going to call `open_dataset` and then `concat` instead. (The difference is that `open_mfdataset` uses dask.)